texar/modules/connectors/connectors.py

```diff
@@ -37,6 +37,8 @@
     "MLPTransformConnector",
     "ReparameterizedStochasticConnector",
     "StochasticConnector",
+    "sum_output_size",
+    "mlp_transform"
```

Comment:
I think these two methods are intended for internal usage only. Are they used somewhere else?

Reply:
I see, I think we can use simpler methods instead, so to keep them internal.

Reply:
These two functions are restored.